tslint.json

```diff
@@ -1,43 +1,43 @@
 {
-  "rules": {
-    "class-name": true,
-    "comment-format": [true, "check-space"],
-    "no-async-without-await": true,
-    "indent": [true, "spaces"],
-    "one-line": [true, "check-open-brace", "check-whitespace"],
-    "no-var-keyword": true,
-    "quotemark": [true, "single", "avoid-escape"],
-    "semicolon": [true, "always", "ignore-bound-class-methods"],
-    "whitespace": [
-      true,
-      "check-branch",
-      "check-decl",
-      "check-operator",
-      "check-module",
-      "check-separator",
-      "check-type"
-    ],
-    "typedef-whitespace": [
-      true,
-      {
-        "call-signature": "nospace",
-        "index-signature": "nospace",
-        "parameter": "nospace",
-        "property-declaration": "nospace",
-        "variable-declaration": "nospace"
-      },
-      {
-        "call-signature": "onespace",
-        "index-signature": "onespace",
-        "parameter": "onespace",
-        "property-declaration": "onespace",
-        "variable-declaration": "onespace"
-      }
-    ],
-    "no-internal-module": true,
-    "no-trailing-whitespace": true,
-    "no-null-keyword": true,
-    "prefer-const": true,
-    "jsdoc-format": true
-  }
+	"rules": {
+		"class-name": true,
+		"comment-format": [true, "check-space"],
+		"no-async-without-await": true,
+		"indent": [false, "tabs"],
```

Comment:
what does "false" do here?

Reply:
make it not do

Reply:
and when not do, linting passes

Reply:
I'll run a chore and update all files to re-enable linting with tabs as the default after this work is released.